SYCL/Assert/assert_in_simultaneously_multiple_tus.cpp

```diff
@@ -0,0 +1,96 @@
+// FIXME unsuppoerted on CUDA until fallback libdevice becomes available
+// UNSUPPORTED: cuda
+// FIXME disable the test on Linux on OpenCL on FPGA as there's some funny
+// failure taking place in this configuration on CI. The test should be enabled
+// with this configuration once it's debugged.
+// UNSUPPORTED: linux && opencl && fpga
```

Comment:
@vladimirlaz does this seem correct way to disable the test only on Linux with OpenCL for FPGA device?

Reply:
yes